crates/nostr-sdk/src/client/mod.rs

```diff
@@ -1413,6 +1413,24 @@ impl Client {
         self.gift_wrap_to(urls, &receiver, rumor, None).await
     }
 
+    /// Comment
+    ///
+    /// <https://github.com/nostr-protocol/nips/blob/master/22.md>
+    #[inline]
+    pub async fn comment<S>(
```

Comment:
I'm thinking to remove all these kind of methods from `Client` to keep the APIs more cleaned. At least, the ones that can be simply constructed directly with the `EventBuilder`:

```rust
let builder = EventBuilder::<name>;
self.send_event_builder(builder).await
```

The more complex ones (the ones that requre more than 2 steps, are dangerous if used incorrectly or are very frequently used) can be left, like `set_metadata`, `gift_wrap`, `send_private_msg`, etc.

This is still an idea. What do you think?
I started removing these methods in PR #608

Reply:
Yes, I prefer using EventBuilder than client's method